code/src/components/Album.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+
+
+export const Album = (props) => {
+    
+ 
+    //return the album cover 
+    return (
+        <>
```

Comment:
For this part you can use in the future an option of 

<React.Fragment>  

the above example can also store a key value as well, when you map before it.